Gifski/GifskiWrapper.swift

```diff
@@ -107,4 +139,12 @@ final class GifskiWrapper {
 	func finish() throws {
 		try wrap { gifski_finish(pointer) }
```

Comment:
I suggest setting the pointer to null ASAP, so that multiple calls can't crash it:

``
var tmp = pointer;
pointer = null;
gifski_finish(tmp);
```

Reply:
I already prevent multiple calls to `finish()`: https://github.com/sindresorhus/Gifski/pull/189/files#diff-1df33fd7967eaf75e9f4c10c855f03f7R153-R155

Reply:
But that's not thread-safe. `gifski_finish()` can take a while, so two calls to `finish()` one after another will call  `gifski_finish()` twice, then set the flag twice.

Reply:
Good point. Done.